src/Symfony/Component/Security/Tests/Http/Session/SessionAuthenticationStrategyTest.php

```diff
@@ -39,6 +39,10 @@ public function testUnsupportedStrategy()
 
     public function testSessionIsMigrated()
     {
+        if (PHP_VERSION_ID >= 50400 && PHP_VERSION_ID < 50411) {
+            $this->markTestSkipped('We must not destroy the old session on php 5.4.0 - 5.4.10.');
```

Comment:
the message should rather be ``cannot``.

and instead of having 2 tests where one is always skipped while the other runs, shouldn't we have a single test with the version check in it ?